app/assets/stylesheets/sufia/_styles.scss

```diff
@@ -75,3 +75,13 @@ label.disabled {
   text-decoration:underline;
   background-color:#0088CC;
 }
+
+.check_all {
+  white-space: nowrap;
+  .caret {
+    margin-left: 0;
+  }
+  input#check_all {
```

Comment:
@jcoyne I assumed we were using a check all box in other places other than here and I didn't want to affect the styling for all of them.  But after looking around it looks like we're not.  So I'll consolidate all the check_all CSS.